.vscode/launch.json

```diff
@@ -24,6 +24,7 @@
             "program": "${workspaceFolder}/gen_vehicle_model.py",
             "console": "internalConsole",
             "args": [
+                "-T sdv_model",
```

Comment:
Why renaming it from "vehicle_model"?

Reply:
Inside the SDK, sdv_model is used and the generated files needs to match the sdv_model for all the internal imports otherwise a manual adaptation is required after the generation.